file/1.3/modules/ROOT/pages/file-documentation.adoc

```diff
@@ -663,11 +663,22 @@ A matcher can also be used for additional filtering of files.
 [[time-between-size-check]]
 === Time Between Size Check Parameter Behavior
 
-This wait is performed at the moment of retrieving the content of the file.
-If the flow never accesses the content of the file, the check is not done. Also,
-because this wait can take place in multiple message processors, this connector has no
-control over which thread will be the one performing the wait, thus leaving the possibility
-open to block threads not meant for this type of behavior.
+The process of the Write operation on a file can take some time to complete and get the written-to file ready to be read or accessed.  Thereby, MuleSoft introduced the `timeBetweenSizeCheck` parameter to prevent an operation from accessing an incomplete file.
```

Comment:
Suggestion to remove unnecessary words and not use passive voice:
"The Write operation on a file can take time to complete and get the file ready to read or access. Use the `timeBetweenSizeCheck` parameter to prevent an operation from accessing an incomplete file.